src/ts/universe.ts

```diff
@@ -174,13 +174,21 @@ export class UniversePlayer extends UniverseCircle {
 
     if (!charge) return;
 
+    canvasContext.save();
+
     canvasContext.fillStyle = '#e43';
+    canvasContext.translate(vector.x, vector.y);
+    canvasContext.rotate(orientation - Math.PI / 2);
```

Comment:
It is easier to do the work here than to use the fillRect function we had. Now we just center the canvas on the player w/ the right orientation, and draw the charge bar offset from the player's center by radius + radius/2